client/trino-jdbc/pom.xml

```diff
@@ -333,6 +338,10 @@
                                     <pattern>io.trino.client</pattern>
                                     <shadedPattern>${shadeBase}.client</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>com.google.errorprone</pattern>
+                                    <shadedPattern>${shadeBase}.errorprone</shadedPattern>
```

Comment:
Should this be part of the build at all?

also, did you mean to remove 

```
<relocation>
                                    <pattern>javax.annotation</pattern>
                                    <shadedPattern>${shadeBase}.javax.annotation</shadedPattern>
                                </relocation>
```
?

Reply:
it's still imported by the transitive dependencies. I can try to exclude it.

Reply:
It's brought by the guava. I've excluded it and it's no longer needed